src/Handler/BindCommandHandler.php

```diff
@@ -65,12 +65,12 @@ public function __construct(DiscoveryManager $discoveryManager, PackageCollectio
     /**
      * Handles the "bind --list" command.
      *
-     * @param Args $args The console arguments.
-     * @param IO   $io   The I/O.
+     * @param Args $args        The console arguments.
+     * @param IO   $inputOutput The I/O.
```

Comment:
Could you revert the `$io` change everywhere? I think the shorter variable name is in fact more readable, since I/O is a very widely known abbreviation. I disabled the corresponding check on Scrutinizer now.